### PR TITLE
[Task]:Artificial intelligence content updates

### PR DESCRIPTION
**Pull request by Alexandra-Green**
## What this PR accomplishes

- new call to action labels on AI resource page + translation
- removes truncation from facet options
- new "Asset type" facet option for AI assets + translation
- new "Asset type" value for AI assets
- new AI asset page alert callout box content + translation

## Issue(s) addressed

- DATA-1260
- DATA-1262
- DATA-1263

## What needs review
Existing algorithm/ai assets will not have this change unless the asset type is changed to the new Artificial Intelligence option. Either switch one to the new option or create a new asset of type AI

Resource page 
- call to action labels on AI resource page + translation matches to ticket DATA-1262
Dataset page
- AI asset page alert callout box content + translation matches to ticket DATA-1260
Search page
- "Asset type" facet option for AI assets + translation matches to ticket DATA-1263
- truncation is removed from facet options.

### CODE REVIEW

**Comment by KatiRG** [comment]
@Alexandra-Green The link in the blue callout boxes leads to a 404 error (in EN and FR):
Screenshot 2023-09-22 at 13 53 03


Screenshot 2023-09-22 at 13 52 32

File: ckanext/ontario_theme/templates/internal/package/read.html [hunk 1]
```diff
@@ -22,22 +22,17 @@
...
-              {% trans url=h.url_for('group.about', id='artificial-intelligence') %}
-                This record describes an algorithm or artificial intelligence asset. It does not describe a dataset.
-                <a href="{{ url }}">
-                  Learn more here about how the Government of Ontario is working to grow our catalogue beyond data for greater
-                  transparency and accountability.
-                </a>
-              {% endtrans %}
+              {{ _('This record describes an <abbr title="Artificial Intelligence">AI</abbr> asset. It does not describe a dataset.') }}
             </p>
+            <a href="{{ h.url_for('group.about', id='artificial-intelligence-and-algorithms') }}">
```
Comment (KatiRG) [inline ckanext/ontario_theme/templates/internal/package/read.html:33]: url should like to `id='artificial-intelligence'`

File: ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po [hunk 2]
```diff
@@ -2525,6 +2525,10 @@ msgstr "Visualisez les données"
 msgid "Access data"
 msgstr "Accédez aux données"
 
+#: ckanext/ontario_theme/templates/internal/scheming/package/resource_read.html:68
+msgid "Access <abbr title=\"Artificial Intelligence\">AI</abbr>"
+msgstr "Accédez à l’<abbr title =\"intelligence artificielle\">IA</abbr>"
```
Comment (KatiRG) [inline ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po:2530]: Should the french title be capitalized?

Comment (Alexandra-Green) [reply ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po:2530]: No and neither should the English because are using sentence case

**Review by KatiRG — APPROVED** [review]

**Pull request merged**